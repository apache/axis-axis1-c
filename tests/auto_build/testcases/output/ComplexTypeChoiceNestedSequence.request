POST /ComplexTypeChoice2/services/Service HTTP/1.1
Host: 127.0.0.1:13260
Content-Type: text/xml; charset=UTF-8
SOAPAction: "http://complextype.test.apache.org"
Content-Length: 448

<?xml version='1.0' encoding='utf-8' ?>
<SOAP-ENV:Envelope xmlns:SOAP-ENV="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
<SOAP-ENV:Body>
<ns1:asComplexTypeRequest xmlns:ns1="http://xsd_byte.test.apache.org">
<SimpleComplexType1><field3>I am still here!</field3>
</SimpleComplexType1>
</ns1:asComplexTypeRequest>
</SOAP-ENV:Body>
</SOAP-ENV:Envelope>

