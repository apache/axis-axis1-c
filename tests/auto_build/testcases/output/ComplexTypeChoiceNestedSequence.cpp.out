Result field3 is = I am still here!
---------------------- TEST COMPLETE -----------------------------
